client/v1/client.go

```diff
@@ -659,6 +660,39 @@ func (c *Client) Do(req *http.Request, result interface{}, codes ...int) (*http.
 	return resp, nil
 }
 
+func (c *Client) Logs(w io.Writer, q map[string]string) error {
```

Comment:
This really isn't too useful since it never returns, unless there is an error.

We should add a context.Context argument so users can cancel the request if they like.

Reply:
fixed.